.github/workflows/build.yml

```diff
@@ -38,11 +73,16 @@ jobs:
     - name: test-code-formatting
       run: |
         ./test.sh test-pylint-check
-        shellcheck ./test.sh
+
+        # Skip shellcheck on MacOSX. It's throwing up some weird errors!
+        if [ "$RUNNER_OS" == "Linux" ]; then
+            shellcheck ./test.sh
+        fi
 
     #! -------------------------------------------------------------------------
     - name: test-pytests
-      run: ./test.sh test-pytests
+      run: |
+        ./test.sh test-pytests
```

Comment:
I'm not familiar with these files, but just checking: is this deliberate?

Reply:
Yes. This is a test-method to drive all python tests. As a user you don't need to bother yourself with `cd` to some other dir. You can just stay at top-level L3 root-dir and do:

`$ ./test.sh test-pytests`